docs/uninstall.md

```diff
@@ -0,0 +1,449 @@
+---
+title: "Uninstalling Knative"
+weight: 85
+type: "docs"
+---
+
+To uninstall an Operator-based Knative installation, follow the [Uninstall an Operator-based Knative Installation](#uninstall-an-operator-based-knative-installation) procedure below.
+To uninstall a YAML-based Knative installation, follow the [Uninstall a YAML-based Knative Installation](#uninstall-a-yaml-based-knative-installation) procedure below.
+
+
+## Uninstalling a YAML-based Knative installation
+
+To uninstall a YAML-based Knative installation:
+
+
+### Uninstalling optional Serving extensions
+
+Uninstall any Serving extensions you have installed by following the relevant steps below:
+
+{{< tabs name="serving_extensions" >}}
+
+{{% tab name="HPA autoscaling" %}}
+
+Knative also supports the use of the Kubernetes Horizontal Pod Autoscaler (HPA) for driving
+autoscaling decisions.
+The following command will install the components needed to support HPA-class autoscaling:
+
+```bash
+kubectl delete -f {{< artifact repo="serving" file="serving-hpa.yaml" >}}
+```
+
+<!-- TODO(https://github.com/knative/docs/issues/2152): Link to a more in-depth guide on HPA-class autoscaling -->
```

Comment:
Probably don't need this for remove

Reply:
I agree regarding the comment. Regarding the command, we spell out why the user might run the command in the header, so I'm okay with letting the reader make that call.